src/libs/sanitizeStringForJSONParse.js

```diff
@@ -0,0 +1,28 @@
+const replacer = str => ({
+    '\\': '\\\\',
+    '\t': '\\t',
+    '\n': '\\n',
+    '\r': '\\r',
+    '\f': '\\f',
+    '"': '\\"',
+}[str]);
+
+/**
+ * Replace any characters in the string that will break JSON.parse for our Git Log output
+ *
+ * Solution partly taken from SO user Gabriel Rodríguez Flores 🙇
+ * https://stackoverflow.com/questions/52789718/how-to-remove-special-characters-before-json-parse-while-file-reading
+ *
+ * @param {String} inputString
+ * @returns {String}
+ */
+function sanitizeStringForJSONParse(inputString) {
+    if (!inputString || typeof inputString !== 'string') {
+        return '';
```

Comment:
NAB but maybe it's better to just throw an error rather than returning an empty string

Reply:
+1